shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -554,7 +555,7 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(dstFile.getAbsolutePath() + randomSuffix);
+    File tmpDst = new File(Name.HOME + randomSuffix);
```

Comment:
@aaudiber  I am talking about the current master branch. If we want to follow the convention, I can do it. You mean, say if the command is `bin/alluxio fs copyToLocal /LICENSE /home/username`, the tmpfile should be`/home/username.sda3dfg35_`? or `/home/username/.sda3dfg35`